guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx

```diff
@@ -0,0 +1,1310 @@
+---
+title: 'How to send PUSH notifications to iOS devices with Firebase Cloud Messaging (FCM) using Novu'
+description: 'Leverage the digest functionality to send email notifications'
+icon: 'circle-2'
+---
+## **Prerequisites**
+
+To complete this tutorial, you will need the following:
+
+- [Apple Developer](https://developer.apple.com/) membership (to obtain the required permissions to send push notifications)
+- Laptop running macOS to develop an iOS project
+- [Firebase](https://firebase.google.com/) account
+- [Novu](https://web.novu.co/) account
+- [Xcode](https://developer.apple.com/xcode/) installed on your Mac
+
+## Create an iOS app
+
+Open Xcode and create a new project, choosing the **App** template.
+
+Give your project a name, and select your own name as the team for now, or leave it blank. 
+
+Your bundle identifier will be generated here — you will use this later when connecting to Firebase. Select **SwiftUI** as your interface and **Swift** as your language.
+
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/1.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/2.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/3.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/4.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/5.png"/> </Frame>
+
+Let's now build and run our ios application and see if everything is working correctly
+
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/6.png"/> </Frame>
+
+Go to the firebase documentation website go to docs right here and select cloud messaging we're going to be following this guide to set up
```

Comment:
select 'Cloud Messaging'

Reply:
Go to the Firebase docs, and select 'Cloud Messaging'. We're going to follow this guide to add cloud messaging to our app.